openshift/template.yaml

```diff
@@ -144,6 +144,9 @@ parameters:
 - name: ENABLE_UPGRADE_AGENT
   value: "false"
   required: false
+- name: INSTALLER_CACHE_SIZE
+  value: "5"
```

Comment:
This strikes me as small, and potentially leading to some cache churn. How many installers can we have before we risk hitting the disk space issue?

Reply:
This number is based on the data we have about the ocp version deviation. It covers 90% of the cases. I attached the graph and a full explanation to the Jira ticket.